csharp-basic-topics/StringBuilderType/Tests/Tests.cs

```diff
@@ -0,0 +1,230 @@
+namespace Tests;
+
+using StringBuilderType;
+using System;
+using Xunit;
+
+public class Tests
+{
+    public static readonly StringBuilderMethods stringBuilderMethods = new StringBuilderMethods();
+    [Fact]
+    public void GivenAText_AppendAndReturnStringBuilder()
+    {
+        string text = "Appending a value to StringBuilder";
+
+        //Act
+        var response = stringBuilderMethods.Append(text);
+
+        //Assert
+        Assert.Equal(text.Length, response.Length);
+    }
+
+    [Fact]
+    public void WithDefaultText_AppendAndReturnStringBuilder()
+    {
+        //Act
+        var response = stringBuilderMethods.Append();
+
+        //Assert
+        Assert.NotNull(response);
+        Assert.InRange(response.Length,0, response.MaxCapacity);
+    }
+
+    [Fact]
+    public void GivenAText_AppendAndReturnAnException_()
+    {
+        string text = "Appending a value to StringBuilder";
+        //Act
+        try
+        {
+            var response = stringBuilderMethods.Append(text, int.MaxValue);
+        }
+        catch (Exception ex)
+        {
+            //Assert
+            Assert.IsType<OutOfMemoryException>(ex);
+        }
```

Comment:
It won't ever go to the catch block so what's the point?
You should try to produce the exception manually to show how it works in another test.